homeassistant/components/spc/__init__.py

```diff
@@ -59,7 +63,12 @@ async def async_upate_callback(spc_object):
 
     hass.data[DATA_API] = spc
 
-    if not await spc.async_load_parameters():
+    for _ in range(0, STARTUP_RETRIES):
```

Comment:
I think it makes more sense to migrate this integration to the UI by implementing a config flow.

The reason for this is that a config entry set up can fail, and Home Assistant will retry set up automatically.

Reply:
I agree that is a better long term solution. At this point I'm just looking for it to start at all. Would this intermediate fix be acceptable as such?

Reply:
Nothing more permanent than a temporary solution :)

There are really just 2 configuration parameters to move into the configuration flow, should be fairly straight forward.

Reply:
I'll take a look